website/docs/cli/commands/completion.mdx

```diff
@@ -28,6 +28,64 @@ When the generated completion script is loaded into the shell, pressing the tab
 Run `atmos completion --help` to see all the available options
 :::
 
+
+## Zsh Completion Setup
```

Comment:
Please use Docusarus tabs. https://docusaurus.io/docs/3.1.1/markdown-features/tabs

Plenty of examples like https://atmos.tools/install

Reply:
Done